Source/Engine/Platform/Base/NetworkBase.cpp

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) 2012-2021 Wojciech Figat. All rights reserved.
+
+#include "NetworkBase.h"
+
+bool NetworkBase::Init()
+{
+    return false;
+}
+
+void NetworkBase::Exit()
+{
+}
+
+bool NetworkBase::CreateSocket(NetworkSocket& socket, NetworkProtocolType proto, NetworkIPVersion ipv)
+{
+    return false;
```

Comment:
All methods in `NetworkBase` should return `true` by default as error of no implementation.